### PR TITLE
Reusable fixture, for reuse in plugins and api

### PR DESCRIPTION
**Pull request by volodymyrss**

### CODE REVIEW

File: cdci_data_analysis/pytest_fixtures.py [hunk 1]
```diff
@@ -0,0 +1,251 @@
...
+        smtp_server: 'localhost'
+        sender_mail: 'team@odahub.io'
+        cc_receivers_mail: ['team@odahub.io']
+        smtp_port: 1025
+        smtp_server_password: ''
+        email_sending_timeout: True
+        email_sending_timeout_default_threshold: 1800
+        email_sending_job_submitted: True
+    """)
+
```
Comment (burnout87) [inline cdci_data_analysis/pytest_fixtures.py:164]: Like this, don't you end up writing within the 'test-dispatcher-conf.yaml' file the same content?

**Review by burnout87 — COMMENTED** [review]
Will this approach completely replace the 'tests/test-conf.yaml' file?

**Comment by volodymyrss** [comment]
> Will this approach completely replace the 'tests/test-conf.yaml' file?

yes

File: cdci_data_analysis/pytest_fixtures.py (hunk 1 above)
Comment (volodymyrss) [reply cdci_data_analysis/pytest_fixtures.py:164]: yes. but it's easier to do like this that to distribute and then discover yet another non-code file. see how discovery of the test*yaml was rather cumbersome

Comment (burnout87) [reply cdci_data_analysis/pytest_fixtures.py:164]: Nice, it will just have to be aligned with the latest version of the configuration

**Review by burnout87 — APPROVED** [review]

**Pull request merged**